src/Talis/Persona/Client/Base.php

```diff
@@ -1,7 +1,10 @@
 <?php
 namespace Talis\Persona\Client;
+require __DIR__ . '/../../../../vendor/autoload.php';
```

Comment:
OK, not sure what pattern is normally used for this. Would you normally call the `autoloader` from `index.php`?